docs/reference/commandline/dockerd.md

```diff
@@ -731,7 +731,7 @@ Also Windows Container makes use of `--exec-opt` for special purpose. Docker use
 can specify default container isolation technology with this, for example:
 
 ```bash
-$ sudo dockerd --exec-opt isolation=hyperv
+$ dockerd --exec-opt isolation=hyperv
```

Comment:
I wonder this should be

  \```powershell
  \> dockerd --exec-opt isolation=hyperv
  \```

(If so, other Windows-specific codes should be updated as well)

cc @jhowardmsft